src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -264,17 +243,17 @@ export class DotEditContentHtmlService {
     }
 
     private setEditMode(): void {
-        this.addContainerToolbar();
-        this.addContentletMarkup();
+        this.generateContentToolBars();
```

Comment:
I'm not sure about the verb _generate_ thoughts @luisrojascr @freddyucv

Reply:
create?

Reply:
I will use create then.